apps/meteor/tests/end-to-end/api/livechat/00-rooms.ts

```diff
@@ -1256,6 +1256,32 @@ describe('LIVECHAT - rooms', () => {
 			await deleteVisitor(visitor.token);
 		});
 
+		it("should set visitor's source as API after uploading a file", async () => {
+			await updateSetting('FileUpload_Enabled', true);
+			await updateSetting('Livechat_fileupload_enabled', true);
```

Comment:
Let's undo these changes `after` all...

Reply:
Isn't it some sort of pattern from our side not to delete visitors and livechat rooms in tests? I recall @KevLehman mentioned something about this some time ago because that's an expensive behavior
None of the tests in this `describe` delete the visitor